modules/Core/includes/endpoints/UpdateUsernameEndpoint.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @param int $id The NamelessMC user to update
+ * @param string $username The new username of the user
+ *
+ * @return string JSON Array
+ */
+class UpdateUsernameEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'updateUsername';
+        $this->_module = 'Core';
+        $this->_description = 'Update a users NamelessMC username to a new username';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.